### PR TITLE
Updated Extensions for date fields - Base Schema Only

### PR DESCRIPTION
**Pull request by TNAemmaPea**
As discussed with Ian
In the base schema properties that will not be validated in the CSV upload have been removed and error codes in the messages so these can be processed later
closure types added to the definitions schema

### CODE REVIEW

File: metadata-schema/baseSchema.schema.json [hunk 1]
```diff
@@ -24,65 +11,45 @@
         }
       ],
       "message": {
```
Comment (TomJKing) [inline metadata-schema/baseSchema.schema.json:13]: I'm not sure I understand how this works. The keys `message` and `format` don't make sense given the it appears that these represent error messages, ie use `ERROR_...` Does it not make more sense to have:
```
...
"errorMessage": "ERROR_..."`
...
```

File: metadata-schema/baseSchema.schema.json [hunk 2]
```diff
@@ -24,65 +11,45 @@
...
-        "format": "Description should be less than 8000 characters"
+        "format": "ERROR_DESCRIPTION_1"
       },
       "alternateKeys": [
         {
           "tdrFileHeader": "Description"
         }
-      ]
+      ],
+      "daDescription": "WARNING_DESCRIPTION_1"
```
Comment (TomJKing) [inline metadata-schema/baseSchema.schema.json:52]: I don't understand how `daDesciption` relates to "warnings". Does it not make more sense to have this as:
...
"warning": "WARNING_DESCRIPTION_1"`

File: metadata-schema/baseSchema.schema.json [hunk 3]
```diff
@@ -298,24 +254,15 @@
...
-      "alternateKeys": [
-        {
-          "tdrFileHeader": "States if the type is a file or folder"
-        }
-      ]
     }
   }
 }
+
+
```
Comment (TomJKing) [inline metadata-schema/baseSchema.schema.json:268]: Unnecessary line break

File: metadata-schema/baseSchema.schema.json [hunk 4]
```diff
@@ -298,24 +254,15 @@
       ],
       "maxLength": 8000,
       "message": {
-        "format": "Description Former Reference should be less than 8000 characters"
+        "format": "ERROR_FORMER_REFERENCE_DEPARTMENT"
```
Comment (TomJKing) [inline metadata-schema/baseSchema.schema.json:257]: For consistency this should be `ERROR_FORMER_REFERENCE_DEPARTMENT_1`

File: metadata-schema/baseSchema.schema.json [hunk 5]
```diff
@@ -91,106 +58,83 @@
...
-    },
-    "parent_reference": {
       "type": [
-        "string",
+        "array",
         "null"
       ],
+      "items": {
+        "type": "string",
+        "enum": [
```
Comment (TomJKing) [inline metadata-schema/baseSchema.schema.json:104]: The accepted language types should do into the "definitions"schema

File: metadata-schema/baseSchema.schema.json (hunk 1 above)
Comment (ian-hoyle) [reply metadata-schema/baseSchema.schema.json:13]: this was was by the json schmea validator for the error message 
after looking at the documentation these can be defined in a separate properties file 
[https://github.com/networknt/json-schema-validator/blob/master/src/main/resources/jsv-messages.properties](https://github.com/networknt/json-schema-validator/blob/master/src/main/resources/jsv-messages.properties)
so removing all messages to make schema cleaner

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

I'm assuming the error / warning messages will be added in a separate PR when appropriate.

**Pull request merged**